draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1021,7 +1020,9 @@ timestamp of the previous update.
 
 `tree_head` contains the latest tree head information (see {{tree_head}}).
 
-`signature` is a signature over the encoded `tree_head` field.
+`signature` is computed over the encoded `tree_head` field,
+computed using the appropriate signature algorithm for the log indicated by
+`algorithm`.
```

Comment:
"algorithm" isn't there any more.

I suggest rewriting this paragraph as:
`signature` is computed over the `tree_head` field using the signature algorithm declared in the log's parameters (see {{log_parameters}}).